bot/exts/moderation/modpings.py

```diff
@@ -126,6 +183,38 @@ async def on_command(self, ctx: Context) -> None:
 
         await ctx.send(f"{Emojis.check_mark} Moderators role has been re-applied.")
 
+    @modpings_group.command(name='schedule')
+    @has_any_role(*MODERATION_ROLES)
+    async def schedule_modpings(self, ctx: Context, start: str, end: str) -> None:
+        """Schedule modpings role to be added at <start> and removed at <end> everyday at UTC time!"""
+        start, end = parse(start), parse(end)
+
+        if end < start:
+            end += datetime.timedelta(days=1)
+
+        if (end - start) < datetime.timedelta(hours=MINIMUM_WORK_LIMIT):
+            await ctx.send(
+                f":x: {ctx.author.mention} You need to have the role on for "
+                f"a minimum of {MINIMUM_WORK_LIMIT} hours!"
+            )
+            return
+
+        start, end = start.replace(tzinfo=None), end.replace(tzinfo=None)
+        work_time = (end - start).total_seconds()
+
+        await self.modpings_schedule.set(ctx.author.id, f"{start.timestamp()}|{work_time}")
+
+        self._modpings_scheduler.schedule_at(
+            start,
+            ctx.author.id,
+            self.add_role_schedule(ctx.author, work_time, start)
+        )
+
+        await ctx.send(
+            f"{Emojis.ok_hand} {ctx.author.mention} Scheduled mod pings from "
+            f"{start: %I:%M%p} to {end: %I:%M%p} UTC Timing!"
+        )
```

Comment:
> Instead of the user having to convert from UTC to their timezone, you can use an embed timestamp.
> 
> [Here](https://github.com/python-discord/bot/blob/main/bot/exts/moderation/modpings.py#L89-L109) is an example.

Yes, but I don't think it is possible to put two timestamps in a single embed.

Reply:
This is unfortunately true.